src/core/dbus-manager.c

```diff
@@ -790,6 +790,8 @@ static int method_generic_unit_operation(
         if (r < 0)
                 return r;
 
+        LOG_CONTEXT_PUSH_UNIT(u);
+
```

Comment:
hmm, this one seems unnecessary now that the individual handlers push the context and bus_unit_validate_load_state doesn't log